Update Argentina geographic area codes and timezone

ciri-cuervo opened the pull request:
- Correntions on Argentina geographic area codes
- Included new geographic area codes in `54.txt`
- Changed timezone from `America/Buenos_Aires` to `America/Argentina/Buenos_Aires` in `map_data.txt`

ciri-cuervo commented:
I have just signed the CLA.

kkeshava commented on resources/geocoding/en/54.txt:
```diff
@@ -13,224 +13,332 @@
+54114858|Autonomous City of Buenos Aires
+54114919|Autonomous City of Buenos Aires
+54220|Merlo, Buenos Aires
+542202|La Matanza, Buenos Aires
```
2202 is given as González Catán , Viceroy of the Pine in wikipedia. La Matanza is not mentioned anywhere in the whole doc.

kkeshava commented on resources/geocoding/en/54.txt:
```diff
@@ -13,224 +13,332 @@
+54220|Merlo, Buenos Aires
+542202|La Matanza, Buenos Aires
+54221|La Plata, Buenos Aires
+542221|Magdalena/Punta Indio, Buenos Aires
```
We had 2221 as Verónica. But you have changed it to Magdalena/Punta Indioa. Could you help us understand why it should be these two only? Why not any other area listed like Verónica or Álvarez Jonte etc?

kkeshava commented on resources/geocoding/en/54.txt:
```diff
@@ -13,224 +13,332 @@
+54221|La Plata, Buenos Aires
+542221|Magdalena/Punta Indio, Buenos Aires
+542223|Brandsen, Buenos Aires
+542224|Almirante Brown, Buenos Aires
```
Again this is given as Glew, Guernica.

ciri-cuervo commented:
@kkeshava thanks for your review. I have decided to remove some of the new values to include them in a new PR with the associated sources. I have also updated some of the cities based on a further research.

All argentina code areas are provided by ENACOM.
https://www.enacom.gob.ar/multimedia/noticias/archivos/201801/archivo_20180118103440_4860.xls


Regarding La Matanza: both González Catán and Virrey del Pino are cities located in La Matanza county. I have reverted my changes to *González Catán/Virrey del Pino, Buenos Aires*.

### Here are the differences compared to Wikipedia list:


> 542221|Magdalena, Buenos Aires               ->      542221|Magdalena/Verónica, Buenos Aires

Verónica is the capital city of Punta Indio county, created in 1994 when Magdalena county was divided.
Both Magdalena and Verónica share the same code.
https://en.wikipedia.org/wiki/Magdalena_Partido
https://en.wikipedia.org/wiki/Punta_Indio_Partido


> 542225|Alejandro Korn, Buenos Aires          ->      542225|San Vicente, Buenos Aires

Glew is a city in San Vicente county and it is not even the capital city (which is San Vicente).
The code belongs to San Vicente county.
https://en.wikipedia.org/wiki/San_Vicente_Partido


> 542229|Juan María Gutiérrez, Buenos Aires    ->      542229|Juan María Gutiérrez/El Pato, Buenos Aires

The code belongs to both Juan María Gutiérrez and El Pato.
https://es.wikipedia.org/wiki/Juan_Mar%C3%ADa_Guti%C3%A9rrez_(Buenos_Aires)
https://es.wikipedia.org/wiki/El_Pato


> 542335|Caleufú, La Pampa                     ->      542335|Realicó/Rancul Dept., La Pampa

Caleufú is a city in Rancul department and it is not even the capital city (which is Parera).
Ingeniero Luiggi (previos value) is a city in Realicó department (which the capital is Realicó).
The code belongs to many cities in both departments.


> 542336|Huinca Renancó, Córdoba               ->      542336|Huinca Renancó/Villa Huidobro, Córdoba

The code belongs to both Huinca Renancó and Villa Huidobro.
https://en.wikipedia.org/wiki/Huinca_Renanc%C3%B3
https://en.wikipedia.org/wiki/Villa_Huidobro


> 542337|América, Buenos Aires                 ->      542337|América/Rivadavia, Buenos Aires

Former name of the city is Rivadavia and it is the capital of Rivadavia county.


> 542394|Tres Lomas, Buenos Aires              ->      542394|Tres Lomas/Salliqueló, Buenos Aires

The code belongs to both Tres Lomas and Salliqueló.
https://en.wikipedia.org/wiki/Tres_Lomas_Partido
https://en.wikipedia.org/wiki/Salliquel%C3%B3


> 54280|Trelew, Chubut                         ->      54280|Trelew/Rawson, Chubut

The code belongs to both Trelew and Rawson.
https://en.wikipedia.org/wiki/Rawson,_Chubut
https://en.wikipedia.org/wiki/Trelew


> 543404|San Carlos Centro, Santa Fe           ->      543404|Las Colonias Dept., Santa Fe

San Carlos Centro is a city in Las Colonias county and it is not even the capital city (which is Esperanza).
The code belongs to Las Colonias department.
https://en.wikipedia.org/wiki/Las_Colonias_Department


> 543467|Cruz Alta, Córdoba                    ->      543467|Cruz Alta, Córdoba / San José de la Esquina, Santa Fe

This code belongs to cities in different provinces.
https://es.wikipedia.org/wiki/Cruz_Alta_(C%C3%B3rdoba)
https://es.wikipedia.org/wiki/San_Jos%C3%A9_de_la_Esquina


> 543734|Presidencia de la Plaza, Chaco        ->      543734|Machagai/Presidencia de la Plaza, Chaco

The code belongs to both Machagai and Presidencia de la Plaza.
https://en.wikipedia.org/wiki/Machagai
https://en.wikipedia.org/wiki/Presidencia_de_la_Plaza_Department



### New values added (source: ENACOM)

Both cities belong to Azul county (542281)

>  54228148|Cacharí, Buenos Aires
> 54228149|Chillar, Buenos Aires

Both cities belong to Tandil county (54249)

> 542494491|María Ignacia (Vela), Buenos Aires
> 542494493|Gardey, Buenos Aires

kkeshava commented on resources/geocoding/es/54.txt:
```diff
@@ -14,225 +14,315 @@
 
```
Please do the same changes as mentioned in English version of the file

kkeshava commented on resources/timezones/map_data.txt:
```diff
@@ -5564,7 +5564,7 @@
 52999|America/Mexico_City
 53|America/Havana
-54|America/Buenos_Aires
+54|America/Argentina/Buenos_Aires
```
We use CLDR data for timezone identifiers and this mapping is working as they specify, so we kindly suggest you file an issue with the CLDR team here: https://unicode.org/cldr/trac/newticket
These are standard timezone representation, so please verify before filing an issue.

ciri-cuervo commented on resources/geocoding/en/54.txt:
```diff
@@ -13,224 +13,315 @@
+542221|Magdalena/Verónica, Buenos Aires
+542223|Brandsen, Buenos Aires
+542224|Glew/Guernica, Buenos Aires
 542225|San Vicente, Buenos Aires
```
Actually Alejandro Korn belongs to San Vicente, and the code 2225 is for San Vicente county, not only Alejandro Korn.

kkeshava commented on resources/geocoding/en/54.txt:
```diff
@@ -13,224 +13,311 @@
 # Translated from es/54.txt using Freebase and Wikipedia.
+# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina
 # Some manual edits were made after the file was generated, according to the
 # ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
```
Remove these 4 lines and please replace it with below lines as below,

\# Historically translated from es/54.txt using Freebase and Wikipedia.
\# Some manual edits were made after the file was generated, according to the
\# ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
\# Recently updated by consulting
\# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina

kkeshava commented on resources/geocoding/es/54.txt:
```diff
@@ -14,225 +14,311 @@
-# http://en.wikipedia.org/wiki/+54 [431727225]
+# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina
 # Some manual edits were made after the file was generated, according to the
 # ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
```
Remove these 4 lines and please replace it with below lines as below,

\# Historically generated from:
\# http://www.cnc.gov.ar/infotecnica/numeracion/indicativosinter.asp (2011-07-11)
\# http://en.wikipedia.org/wiki/+54 [431727225]
\# Some manual edits were made after the file was generated, according to the
\# ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
\# Recently updated by consulting 
\# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina

ciri-cuervo commented on resources/geocoding/es/54.txt:
```diff
@@ -14,225 +14,311 @@
-542345|General Pueyrredón
-542346|Chivilcoy
-542352|Chacabuco
+542335|Dpto. Realicó/Rancul, La Pampa
```
in Spanish it is Departamento Realicó, or Departamento Rancul.. but in English it is Realicó Department or Rancul Department.

kkeshava replied:
Oh ok thanks for the clarification

kkeshava commented on resources/geocoding/en/54.txt:
```diff
@@ -12,225 +12,313 @@
 # ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
+# Recently updated by consulting
+# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina
 
```
Last comment, please add below lines as well.

\# 2335 belongs to Caleufú, but it is a city in Rancul department and Ingeniero
\# Luiggi is a city in Realicó department. 3404 belongs to San Carlos Centro a
\# city in Las Colonias.

ciri-cuervo replied:
done

kkeshava commented on resources/geocoding/es/54.txt:
```diff
@@ -12,227 +12,315 @@
 # ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
+# Recently updated by consulting
+# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina
 
```
Please add these lines.

\# 2335 belongs to Caleufú, but it is a city in Rancul department and Ingeniero
\# Luiggi is a city in Realicó department. 3404 belongs to San Carlos Centro a
\# city in Las Colonias.

ciri-cuervo replied:
done

kkeshava commented:
Hi,
Thank you for contributing to libphonenumber! The changes look good to us :)

This PR is approved now. Please update your branch and once it is done successfully, please click on 'Squash and merge'.

ciri-cuervo commented:
@kkeshava thanks for your review.

the branch is updated but I can't merge since it says: *Only those with write access to this repository can merge pull requests*
